src/types/associatedArtifact.types.ts

```diff
@@ -14,14 +14,16 @@ export interface FileRrs {
   configuredType?: CustomerRrsType;
 }
 
+export type PublisherVersion = 'PublishedVersion' | 'AcceptedVersion' | null;
```

Comment:
De lærde strides om sånt helst skal være `type` eller `enum`. Syns `type` ofte er bittelitt enklere å forholde seg til, men åpen for innspill her

Reply:
gjorde om til enum her også, ble lettere å gjenbruke på validering bl.a.